mu/device.py

```
@@ -134,6 +134,60 @@ def backup(self) -> bool:
+                'error',
+                self.name,
+                f'export failed: {e}',
+                stdout=True,
+            )
+            return False
+        self.conf.backup_dir.mkdir(parents=True, exist_ok=True)
+        export_path = self.conf.backup_dir / export_file_name
+        try:
+            export_path.write_text('\n'.join(lines) + '\n')
+            export_path.chmod(0o600)
+        except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)